ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -195,6 +192,7 @@
     <Compile Include="YahooDownloader\YahooDataDownloader.cs" />
     <Compile Include="CsvDataProcessor.cs" />
     <Compile Include="ZipStreamProvider.cs" />
+    <Compile Include="CryptoiqDownloader\Program.cs" />
```

Comment:
Same here; looks like this could be moved back to where it was originally to avoid unnecessary churn.